### PR TITLE
Add support for exporting policy in non Xml encoded format

### PR DESCRIPTION
**Pull request by solankisamir**
- Based on REST Spec PR https://github.com/Azure/azure-rest-api-specs/pull/5763
- added support for Exporting Policies as RAW XML

### CODE REVIEW

**Comment by solankisamir** [comment]
@dsgouda ping ! on this.

File: src/SDKs/ApiManagement/Management.ApiManagement/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@
 [assembly: AssemblyDescription("Provides Api management capabilities for Microsoft Azure.")]
 
 [assembly: AssemblyVersion("4.0.0.0")]
-[assembly: AssemblyFileVersion("4.8.0.0")]
```
Comment (dsgouda) [inline src/SDKs/ApiManagement/Management.ApiManagement/Properties/AssemblyInfo.cs]: Please update the same in the csproj and update packagereleasenotes

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/ApiManagement/Management.ApiManagement/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (solankisamir) [reply src/SDKs/ApiManagement/Management.ApiManagement/Properties/AssemblyInfo.cs]: updated

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge once CIs pass

**Pull request merged**